salt/spm/__init__.py

```diff
@@ -491,10 +494,20 @@ def _install_indv_pkg(self, pkg_name, pkg_file):
 
         # No defaults for this in config.py; default to the current running
         # user and group
-        uid = self.opts.get('spm_uid', os.getuid())
-        gid = self.opts.get('spm_gid', os.getgid())
-        uname = pwd.getpwuid(uid)[0]
-        gname = grp.getgrgid(gid)[0]
+        import salt.utils
+        if salt.utils.is_windows():
+            import salt.utils.win_functions
+            cur_user = salt.utils.win_functions.get_current_user()
+            cur_user_sid = salt.utils.win_functions.get_sid_from_name(cur_user)
+            uid = self.opts.get('spm_uid', cur_user_sid)
+            gid = self.opts.get('spm_gid', cur_user_sid)
+            uname = cur_user
```

Comment:
This is a lot of unnecessary assignment here. Why aren't we just calling this variable `uname` up in L500?